README.md

```diff
@@ -1,2 +1,180 @@
 # mlboard
 mlpack visualization tool
+
+mlboard is a tool that allows you to log mlpack data in the format that the TensorBoard frontend can render in browsers.
+
+Mlboard is still in the development phase. You can track the development ideas [here](https://www.mlpack.org/gsocblog/Jeffin2020CBP.html)
+
+### 0. Contents
+
+  1. [Building mlboard from source](#1-building-mlboard-from-source)
+  2. [Usage](#2-usage)
+
+### 1. Building the source code 
+
+For mlboard to run, you have to install protobuf
+
+- For Mac OS 
+
+`brew install protobuf`
```

Comment:
I think a code block better over here. you can insert a codeblock using ``` twice one before and after the code.
```
brew install protobuf
```

instead of 
`brew install protobuf`